lib/rubygems/test_case.rb

```diff
@@ -1534,24 +1534,4 @@ def self.key_path(key_name)
 
 end
 
-# require dependencies that are not discoverable once GEM_HOME and GEM_PATH
-# are wiped
-begin
-  gem 'rake'
-rescue Gem::LoadError
-end
-
-begin
-  require 'rake/packagetask'
-rescue LoadError
-end
-
-begin
```

Comment:
What this removal have to do with rake?

Reply:
Nothing, they were other unnecessary gem activations and went ahead and removed them too. I'll remove this part from the PR.

Reply:
I was just curious, no big deal

Reply:
Ok, thanks! I'll merge this as is then!